src/components/modals/CreateAntModal/CreateAntModal.tsx

```diff
@@ -380,60 +383,68 @@ function CreateAntModal({ show }: { show: boolean }) {
                                   return (
                                     <>
                                       {/* TODO: add label for mobile view */}
-                                      <ValidationInput
-                                        inputId={row.attribute}
-                                        inputCustomStyle={{
-                                          width: '90%',
-                                          fontSize: '16px',
-                                          background:
-                                            editingField === row.attribute
-                                              ? 'white'
-                                              : 'transparent',
-                                          border: 'none',
-                                          borderRadius: '2px',
-                                          color:
-                                            editingField === row.attribute
-                                              ? 'black'
-                                              : 'white',
-                                          padding:
-                                            editingField === row.attribute
-                                              ? '10px '
-                                              : '10px 0px',
-                                          display: 'block',
+                                      {/* eslint-disable-next-line */}
+                                      <label
+                                        onClick={() => {
+                                          setEditingField(row.attribute);
+                                          setModifiedValue(row.value);
```

Comment:
why are we wrapping in a label?

Reply:
I noticed the "add label for mobile view" note, figured the logic for this was better set up as a label, also gives us a container to control the validationList

Reply:
added onClick to the Validation input wrapper, cant do it on the input because the way we have it setup, the input is disable, which disables the events on it too - such as on click and on focus.